manifests/darwin.pp

```diff
@@ -0,0 +1,62 @@
+# == Class: python::darwin
+#
+# Performs the setup necessary (e.g., downloading the installation MSI) to
+# install Python.  Also sets variables (like $site_packages, $interpreter)
+# that have to be specially calculated on Darwin.
+#
+# === Parameters
+#
+# [*arch*]
+#  The architecture of Python to install, defaults to the architecture of
+#  the system (e.g., 'x64' on 64-bit system).
+#
+# [*allusers*]
+#  Whether to install Python for all users, defaults to true.
+#
+# [*base_url*]
+#  The base url to use when downloading Python, undefined by default.
+#
+# [*source*]
+#  The HTTP or UNC source to the Python package, undefined by default.
+#
+# [*targetdir*]
+#  The target installation directory to use for the Python package,
+#  undefined by default.
+#
+# [*version*]
+#  The version of Python to install, defaults to '2.7.6'.
+#
+# [*darwin_path*]
+#  Whether or not to add Python directory to the Darwin system %Path%,
+#  defaults to true.
+#
+class python::darwin(
+  $allusers  = true,
+  $arch      = $::architecture,
+  $base_url  = undef,
+  $source    = undef,
+  $targetdir = undef,
+  $version   = $python::params::full_version,
+  $darwin_path  = true,
+) inherits python::params {
+
+  $basename = "python-${version}-macosx10.6.pkg"
+
+  $package = "python-${version}-macosx10.6"
+
+  if $base_url {
+      $source_uri = "${base_url}${basename}"
+    } else {
```

Comment:
Or is just a two space convention instead of four that you're interpreting
as an error?

On Tue, Aug 18, 2015, 22:19 Amos Shapira <notifications@github.com> wrote:

> In manifests/darwin.pp
> <https://github.com/counsyl/puppet-python/pull/3#discussion_r37374466>:
>
> > +  $allusers  = true,
> > +  $arch      = $::architecture,
> > +  $base_url  = undef,
> > +  $source    = undef,
> > +  $targetdir = undef,
> > +  $version   = $python::params::full_version,
> > +  $darwin_path  = true,
> > +) inherits python::params {
> > +
> > +  $basename = "python-${version}-macosx10.6.pkg"
> > +
> > +  $package = "python-${version}-macosx10.6"
> > +
> > +  if $base_url {
> > +      $source_uri = "${base_url}${basename}"
> > +    } else {
>
> Indentation error.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/counsyl/puppet-python/pull/3/files#r37374466>.
>